lib/common/pool.h

```diff
@@ -16,6 +16,7 @@ extern "C" {
 
 
 #include <stddef.h>   /* size_t */
+#include "zstd_internal.h"   /* ZSTD_customMem */
```

Comment:
Are you comfortable with having `pool` depending on `zstd_internal` now ?

Reply:
Its not a problem for me, if we ever need to extract it, we just need to delete the advanced versions.